src/pfe/Dockerfile_x86_64

```diff
@@ -159,6 +159,9 @@ RUN chmod -R +rx /file-watcher/idc
 
 WORKDIR /codewind-workspace
 
+WORKDIR /extensions
```

Comment:
since you used absolute path below I don't think the `WORKDIR` change here is necessary